runtime/src/bank.rs

```diff
@@ -581,22 +592,28 @@ impl Bank {
     pub fn deposit(&self, pubkey: &Pubkey, tokens: u64) {
         let mut account = self.get_account(pubkey).unwrap_or_default();
         account.tokens += tokens;
-        self.accounts.store_slow(self.is_root(), pubkey, &account);
+        self.accounts()
+            .store_slow(self.id, self.is_root(), pubkey, &account);
+    }
+
+    fn accounts(&self) -> Arc<Accounts> {
+        if let Some(accounts) = &self.accounts {
+            accounts.clone()
```

Comment:
That looks very expensive. Are you sure you need to do this?

Reply:
It just takes another ref to the Arc, no?

Reply:
Yes, you're right. My mistake.